build_tools/rocksdb-lego-determinator

```diff
@@ -143,14 +143,14 @@ PARALLEL_UNIT_TEST_COMMANDS="[
             $CLEANUP_ENV,
             {
                 'name':'Build and test RocksDB debug version',
-                'shell':'$DEBUG make -j$(nproc) all && $SHM make check > /dev/null 2>&1 || $CONTRUN_NAME=punit_check $TASK_CREATION_TOOL || cat t/log-*',
+                'shell':'$DEBUG make -j$(nproc) all && ($SHM make check > /dev/null 2>&1 || $CONTRUN_NAME=punit_check $TASK_CREATION_TOOL) || cat t/log-*',
                 'user':'root',
                 $PARSER
             },
             $CLEANUP_ENV,
             {
                 'name':'Build and test RocksDB debug version under gcc-4.8.1',
```

Comment:
@siying you mean deleting the entire PARALLEL_UNIT_TEST_COMMANDS section or only the 2nd part that compiles with 4.8.1.?